src/Stratis.Bitcoin/Configuration/Logging/LoggingConfiguration.cs

```diff
@@ -29,36 +28,42 @@ public class ExtendedLoggerFactory : LoggerFactory
 
         public static ILoggerFactory Create()
         {
+            return new CustomLoggerFactory();
+            /*
```

Comment:
This code is commented out, should this be removed?

Reply:
The commented code should probably be re-introduced in some form.

Reply:
Resolved.